python/pyspark/sql/session.py

```diff
@@ -104,6 +104,38 @@ def toDF(self, schema=None, sampleRatio=None):
     RDD.toDF = toDF  # type: ignore[assignment]
 
 
+# TODO: This method can be dropped once support for Python 3.8 is dropped
```

Comment:
Could you make it as an IDed TODO, @FurcyPin ?
For example, we recommend to use a pattern `TODO(SPARK-XXX)` with a concrete JIRA ID.

Reply:
Sure. Done.
https://issues.apache.org/jira/browse/SPARK-38912